src/window.js

```diff
@@ -73,8 +74,8 @@ export default function Window({ application, file, text, type, readonly }) {
     parameter_type: null,
   });
   action_save.connect("activate", () => {
+    if (isEmptyCommitMessage(editor.buffer.text, editor.comment_prefix)) return
```

Comment:
This doesn't work:

1. `./re.sonny.Commit --readonly test/fixtures/with-body/COMMIT_EDITMSG`
2. Delete the commit title/description
3. `isEmptyCommitMessage` returns false

Please test your code, it's a lot of work for maintainers to fix regressions.

`editor.comment_prefix` is not a thing - but you can retrieve it from `parsed`

Reply:
I was testing this on a testing git repo I created and it worked there, is there a reason why it fails on the test file?

Reply:
I see, Perhaps your test file didn't have git comments?

Reply:
Yeah it doesn't, although just to be sure, should `isEmptyCommitMessage` be true or false when it looks like this
![image](https://user-images.githubusercontent.com/60044824/193408252-e92bd51a-f97f-449d-92b9-e529c99f5030.png)

Reply:
in that case `isEmptyCommitMessage` should be false.

Reply:
I updated the checks to use `parsed.comment_prefix` and it properly works on my end now